README.md

```diff
@@ -1,40 +1,161 @@
+
+- _Which movies generate the highest revenue? Is this revenue driven by popularity or late fees?_
+
+![Revenue Popularity 1](artifacts/rev_pop_1_1.png)
+
+The dashboard above present a breakdown of revenue by late fees and regular revenue for each MPAA rating, along with a deeper analysis of the films generating the top 1% of revenue and those ranked No. 1 by total revenue or popularity. Key observations include:
+
+- PG-13 and NC-17 movies generate the highest overall revenue.
+- Among the films generating the top 1% of revenue, there is a negative correlation between late fees and regular revenue, indicating that films earning higher late fees tend to have lower regular rental revenue.
+- There is also a negative correlation between popularity and total revenue for the top-ranking films, suggesting that the most popular films aren't necessarily the most profitable.
+
+![Revenue Popularity 2](artifacts/rev_pop_1_2.png)
+
+When further filtering for PG-13 and NC-17 movies alone, the same negative correlations persist among the top revenue-earning and top-ranking films. This highlights an interesting dynamic where popularity does not always align with profitability, and late fees play a significant role in the revenue of top-grossing films.
+
+- _Which movie categories have the highest popularity and revenue?_
+
+![Revenue Popularity 3]()
+
+The bar chart visualizes rental volumes across various categories, highlighting the discrepancy between categories that generate the most revenue and those that are the most popular by rental volume.
+
+- Most popular categories (by rental volume): (1) Sports, (2) Animation, (3) Action
+- Most lucrative categories (by revenue): (1) Sports, (2) Sci-Fi, (3) Drama
+
+This misalignment suggests that certain categories, despite not having the highest rental volumes, are more profitable due to higher rental rates or increased late fees.
+
+- _Is there any correlation between a movie's length and its popularity, or is it largely influenced by the rental rate?_
+
+![Revenue Popularity 4]()
+
+The bar chart illustrates the distribution of rental volumes across different film lengths and rental rates, uncovering distinct customer behaviors tied to pricing:
+
+- Films with the lowest rental rate show a relatively even distribution across most film lengths, with the exception of films lasting 180 minutes, which have noticeably fewer rentals. This indicates that customers are generally less selective about film length when paying a lower rate, as affordability seems to be the primary factor driving their rental choices.
+- Films with higher rental rates, on the other hand, exhibit a more uneven distribution of rentals, with longer films being rented more frequently. This suggests that customers paying a premium rate are more selective and tend to prefer longer films, likely seeking to maximize their viewing experience.
+
```

Comment:
![Revenue Popularity 5](artifacts/rev_pop_3_2.png)

Reply:
i think its a bit confusing with the 3 charts? it took me a min to realize the it was filtering by rental rates. just the first image should be sufficient i.e., rev_pop_3_1

Reply:
okay let's do that!